docs/job-server.md

```diff
@@ -4,9 +4,14 @@ To run code for real in the production environment, use the [https://jobs.opensa
 Here you can see (even without a login) all the ongoing projects within OpenSAFELY, and the specific _jobs_ that have been run on the server.
 To submit job-requests (i.e., to run one or more actions), the general process is as follows:
 
-* **Log in** using your GitHub credentials (this should happen automatically if you have access to the OpenSAFELY GitHub organisation).
-* **Create a workspace** (or select an existing workspace):
+* **Log in** using your GitHub credentials
+* **Create a project** (or select an existing one):
     * Click the `Add a New WorkSpace` button.
+    * Click the `Register Project` button.
+    * Choose a name, this will be the public facing name for your project.
+    * Click `Save`.
+* **Create a workspace**:
+    * Click the `Create a WorkSpace` button.
```

Comment:
nit: I'd probably fix up the existing "WorkSpace" in "Add a New WorkSpace" and use "Create a Workspace" here, if that's how the word appears on job-server (which I think it does from a quick look).